MovieDatabase/MovieDatabase/Repositories/MovieRepository.cs

```diff
@@ -30,13 +30,12 @@ public List<Movie> GetMovies()
         public Movie GetMovieById(int id)
         {
             return _context.Movies
-                .Where(m => m.Id == id)
```

Comment:
why this change? having the where statement separate from having the expression in the first or default is much cleaner, better and improve readability

Reply:
do this for all